lib/msf/core/modules/loader/base.rb

```diff
@@ -467,6 +467,9 @@ def module_path?(path)
 
   # Tries to determine if a file might be executable,
   def script_path?(path)
+    if path.to_s().end_with?("jar")
+      return true;
+    end
```

Comment:
* This is a guard clause, and can use an end-of-line conditional statement
* `()` is unnecessary when no arguments are provided.
* semicolons are not required
* single quotes are preferred when string interpolation is not required.

```suggestion
    return true if path.end_with?('jar')

```